parser/v2/elementparser.go

```diff
@@ -259,6 +259,52 @@ var expressionAttributeParser = parse.Func(func(pi *parse.Input) (attr Expressio
 	return attr, true, nil
 })
 
+var spreadAttributesParser = parse.Func(
+	func(pi *parse.Input) (attr SpreadAttributes, ok bool, err error) {
+		start := pi.Index()
+
+		// Optional whitespace leader.
+		if _, ok, err = parse.OptionalWhitespace.Parse(pi); err != nil || !ok {
+			return
+		}
+
+		// {
+		if _, ok, err = parse.Or(parse.String("{ "), parse.String("{")).Parse(pi); err != nil ||
+			!ok {
+			pi.Seek(start)
+			return
+		}
+
+		// Expression.
+		if attr.Expression, ok, err = exp.Parse(pi); err != nil || !ok {
```

Comment:
Potential use case for a `parser.Until`, will prevent you from having to trim as below.

Reply:
It may not work in this case actually. But worth thinking about!

Reply:
> Potential use case for a `parser.Until`, will prevent you from having to trim as below.

This can't work here because the expression parser grab the three dots.

The only thing I can do is try to change the "expression parser" but the issue is if I remove the ellipses from the value.

I can't after in the `spreadAttributesParser` check if In my expression I got my ellipses since I need to look into the past.

I think for now the simplest thing is as it its now. *(Note, I try things before saying that)*